tests/assets/plugins/taggers/plugin_trainable_tagger.py

```diff
@@ -117,11 +121,8 @@ class TestTrainableTaggerPlugin(TrainableTagger):
     def __init__(self, client: Client, config: Dict[str, Any] = None):
         super().__init__(client, config)
 
-    class EmptyConfig(Config):
-        pass
-
     def config_cls(self) -> Type[Config]:
-        return self.EmptyConfig
+        return EmptyConfig
```

Comment:
I'm indifferent to them in Python.  I moved it out because the model class above it now needed to use it as well, so it was purely read-order within the file.  Let me know if there is a different style preference; I don't have a strong opinion.